libanswers/custom-header.html

```diff
@@ -37,11 +37,10 @@ <h3 class="heading-col">Start here</h3>
           <h3 class="heading-col">Also try</h3>
           <ul class="list-unbulleted">
             <li><a href="https://libraries.mit.edu/fulltext">FullText Finder <span class="about">Find specific citations</span></a></li>
-            <li><a href="http://libguides.mit.edu/google/googlescholar">Google Scholar for MIT <span class="about">Change settings to get better access</span></a></li>
-            <li><a href="https://libraries.mit.edu/libx">LibX <span class="about">Search/authenticate via browser</span></a></li>
-            <li><a href="http://dspace.mit.edu/">DSpace@MIT <span class="about">MIT research</span></a></li>
-            <li><a href="http://dome.mit.edu/">Dome <span class="about">MIT-digitized images, maps, etc.</span></a></li>
-            <li><a href="https://libraries.mit.edu/about/site-search">Site search</a></li>
+            <li><a href="https://libraries.mit.edu/google-scholar-tips">Google Scholar for MIT <span class="about">Change settings to get better access</span></a></li>
```

Comment:
This is actually the best link for this resource: https://libguides.mit.edu/google/googlescholar

Reply:
I thought we were wanting to use shortcodes for things like this? The link I used forwards the user on to that URL. I'm happy to use either one, personally.

Reply:
Doh, you are right. This guide was recently updated from having the GS settings content being a single page on the Google guide to being the only page on the Google guide. I thought you were changing it to point to the old page on the guide, I somehow missed this was a shortcode. So go ahead with it as planned, my mistake!